docs/source/getting-started/installation.rst

```diff
@@ -22,7 +22,9 @@ only have access to certain clouds, use any combination of
 :code:`"[aws,azure,gcp]"` (e.g., :code:`"[aws,gcp]"`) to reduce the
```

Comment:
With the release planned, maybe lets add lambda here and after L16.
```suggestion
:code:`"[aws,azure,gcp, lambda]"` (e.g., :code:`"[aws,gcp]"`) to reduce the
```

Reply:
nit: no extra space

Reply:
Sounds good!

Reply:
Updated!